.bazelrc

```diff
@@ -148,3 +148,5 @@ try-import .bazelrc.user
 # Enable runfiles even on Windows.
 # Architect resolves output files from data files, and this isn't possible without runfile support.
 test --enable_runfiles
+
+build --experimental_remote_merkle_tree_cache
```

Comment:
This caches the analysis of bazel actions better. Because we have `exports_directories_only = False` (which `ts_library` requires?) every single file in the `node_moduels` gets traversed instead of only the package folders which creates a giant tree. This is also the reason for increasing the circleci `resource_class`.